pkg/agent/agent.go

```diff
@@ -1141,3 +1104,82 @@ func (i *Initializer) patchNodeAnnotations(nodeName, key string, value interface
 func (i *Initializer) getNodeInterfaceFromIP(nodeIPs *utilip.DualStackIPs) (v4IPNet *net.IPNet, v6IPNet *net.IPNet, iface *net.Interface, err error) {
 	return getIPNetDeviceFromIP(nodeIPs, sets.NewString(i.hostGateway))
 }
+
+func (i *Initializer) initLocalConfig() error {
+	nodeName, err := env.GetNodeName()
+	if err != nil {
+		return err
+	}
+	if !i.enableExternalNode {
+		err := i.initNodeLocalConfig(nodeName)
+		if err != nil {
+			return err
+		}
+		i.networkConfig.IPv4Enabled = config.IsIPv4Enabled(i.nodeConfig, i.networkConfig.TrafficEncapMode)
```

Comment:
Why these two lines are not in initK8sNodeLocalConfig()?

Reply:
They were originally in initNodeLocalConfig(), and you also set networkConfig in initVMLocalConfig(). At least we should make it consistent between K8sNode and VM.

Reply:
Make sence, updated.